Applications/MaterialAssist/Contracts/Edgebands/Interfaces/IMaterialAssistClientEdgebands.cs

```diff
@@ -0,0 +1,157 @@
+        /// </summary>
+        /// <param name="workplace"></param>
+        /// <returns></returns>
+        Task<IEnumerable<StorageLocation>> GetStorageLocations(string workplace);
+
+        #endregion Read
+
+        #region Update
+
+        /// <summary>
+        /// Updates the edgeband entity dimensions by id (#).
+        /// </summary>
+        /// <param name="id"></param>
+        /// <param name="length"></param>
+        /// <param name="currentThickness"></param>
+        /// <returns></returns>
+        Task UpdateEdgebandEntityDimensions(string id, double length, double currentThickness);
+
+        /// <summary>
+        /// Updates the edgeband entity comments by id (#).
+        /// </summary>
+        /// <param name="id"></param>
+        /// <param name="comments"></param>
+        /// <returns></returns>
+        Task UpdateEdgebandEntityComments(string id, string comments);
+
+        /// <summary>
+        /// Adds the edgeband entity to storage.
+        /// </summary>
+        /// <param name="id"></param>
+        /// <param name="storageLocation"></param>
+        /// <param name="length"></param>
+        /// <returns></returns>
+        Task StoreEdgebandEntity(string id, StorageLocation storageLocation, double length);
+
+        /// <summary>
+        /// Removes the edgeband entity from storage. Available for all ManagementTypes.
+        /// </summary>
+        /// <param name="id"></param>
+        /// <param name="deleteFromInventory"></param>
+        /// <returns></returns>
+        Task RemoveAllEdgebandEntitiesFromWorkplace(string id, bool deleteFromInventory = false);
+
+        /// <summary>
+        /// Remove a subset of edgeband entities from storage. Only available for ManagementTypes GoodsInStock and Stack.
+        /// </summary>
+        /// <param name="id"></param>
+        /// <param name="quantity"></param>
+        /// <param name="deleteFromInventory"></param>
+        /// <returns></returns>
+        Task RemoveSubsetEdgebandEntitiesFromWorkplace(string id, int quantity, bool deleteFromInventory = false);
+
+        /// <summary>
+        /// Removes single edgeband entity / entities from storage. Only available for ManagementTypes GoodsInStock and Stack.
+        /// </summary>
+        /// <param name="id"></param>
+        /// <param name="quantity"></param>
+        /// <param name="deleteFromInventory"></param>
+        /// <returns></returns>
+        Task RemoveSingleEdgebandEntitiesFromWorkplace(string id, int quantity, bool deleteFromInventory = false);
+
+        #endregion Update
+
+        #region Create
+
+        /// <summary>
+        /// Adds a new edgeband entity.
+        /// </summary>
+        /// <param name="edgebandTypeCode"></param>
+        /// <param name="edgebandEntity"></param>
+        /// <returns></returns>
+        Task<EdgebandEntity> CreateEdgebandEntity(string edgebandTypeCode, EdgebandEntity edgebandEntity);
```

Comment:
this should also be a request object

Reply:
this is a concept, we don´t have the request object yet

Reply:
removed